src/include/api/feature_vector_array.h

```diff
@@ -83,8 +83,13 @@ class FeatureVectorArray {
       const tiledb::Context& ctx,
       const std::string& uri,
       const std::string& ids_uri = "",
-      size_t num_vectors = 0) {
-    auto array = tiledb_helpers::open_array(tdb_func__, ctx, uri, TILEDB_READ);
+      size_t num_vectors = 0,
+      size_t timestamp = 0) {
```

Comment:
Can we support a `TemporalPolicy` argument?

Reply:
Yep I plan to instead of this, but I was planning to do the migration all at once b/c there are many places we need to update. Is that okay with you?

Reply:
sgtm